src/components/TokenBox.js

```diff
@@ -12,12 +12,13 @@ import {
 
 import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
 import { faSortDown } from '@fortawesome/free-solid-svg-icons';
+import { COLORS } from '../styles/themes';
 
 const TokenBox = (props) => (
   <TouchableWithoutFeedback onPress={props.onPress}>
     <View style={styles.wrapper}>
       <Text style={styles.label}>{props.label}</Text>
-      <FontAwesomeIcon icon={faSortDown} color='rgba(0, 0, 0, 0.7)' style={{ marginBottom: 5 }} />
+      <FontAwesomeIcon icon={faSortDown} color={COLORS.textColorShadowDark} style={{ marginBottom: 5 }} />
```

Comment:
Change from `0.7` opacity to `0.8` opacity

Reply:
Reverted to `0.7` opacity on [a6b16fd](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/327/commits/a6b16fdf489a4652038ac614da23fe5485e62020)